test/e2e/webhook_e2e_test.go

```diff
@@ -30,39 +31,34 @@ const (
 )
 
 var _ = Describe("CSVs with a Webhook", func() {
-	var c operatorclient.ClientInterface
-	var crc versioned.Interface
-	var namespace *corev1.Namespace
-	var nsCleanupFunc cleanupFunc
-	var nsLabels map[string]string
+
+	var (
+		generatedNamespace corev1.Namespace
+		c                  operatorclient.ClientInterface
+		crc                versioned.Interface
+		nsLabels           map[string]string
+	)
+
 	BeforeEach(func() {
 		c = newKubeClient()
 		crc = newCRClient()
-		nsLabels = map[string]string{
-			"foo": "bar",
-		}
-		namespace = &corev1.Namespace{
+		generatedNamespace = corev1.Namespace{
```

Comment:
Any idea whether we can add support for injecting labels into the testing namespace generation helper function?

Reply:
With my options support! XDDD